wandb/apis/public/runs.py

```diff
@@ -754,10 +754,54 @@ def scan_history(self, keys=None, page_size=1000, min_step=None, max_step=None):
 
     @normalize_exceptions
     def logged_artifacts(self, per_page=100):
+        """Return an iterable collection of all artifacts logged by this run.
+
+        This method retrieves all output artifacts that were logged during the run.
+
+        Arguments:
```

Comment:
```suggestion
        Args:
```

Reply:
@kptkin The `Run` class functions all use the word "Arguments", hence the usage here. Changing it to `Args` will make this inconsistent from the whole file.

Reply:
that's fine, we want to align to the google docstring style, so let's do the correct ones for the new ones we are adding